js/state/siteSettings.js

```diff
@@ -81,6 +87,18 @@ module.exports.activeSettings = (siteSettings, appState, appConfig) => {
     return defaults.adControl
   })()
 
+  settings.safeBrowsingControl = (() => {
+    if (settings.shieldsUp === false) {
```

Comment:
@diracdeltas should safe browsing turn off with shields? Safe browsing seems independent of that to me.

Reply:
Updated!